assets/js/script.js

```diff
@@ -0,0 +1,47 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+const find = document.querySelector('.find');
+const temperature = document.querySelector('.temperature');
+const city = document.querySelector('.city');
+const API_KEY = 'dcb0cec0612e417fa51142150191005';
+const URL = 'https://api.apixu.com/v1/current.json?';
+const results = document.getElementById('results');
+     // Maps access token goes here
+    var key = 'pk.a5c3fbf2119bfb2275b62eddbccd76b3';
```

Comment:
Why API_KEY and URL with upper case and `key` declared with `var` and with lower case?
` const KEY = `